Tests/DataPipeline/Config/SDKConfigBuilderTest.swift

```diff
@@ -78,7 +78,11 @@ class SDKConfigBuilderTest: UnitTest {
         let analyticsConfig = Configuration(writeKey: givenCdpApiKey).values
 
         var analyticsDefaultExpectedSettings = Settings(writeKey: givenCdpApiKey)
-        analyticsDefaultExpectedSettings.integrations = try! JSON(["Segment.io": true])
+        do {
+            analyticsDefaultExpectedSettings.integrations = try JSON(["Segment.io": true])
+        } catch {
+            XCTFail("Failed to create JSON with error: \(error)")
```

Comment:
Might be more descriptive slightly

```suggestion
            XCTFail("Failed to setup test integrations with error: \(error)")
```